test/Microsoft.Health.Dicom.Tests.Integration/Features/RetrieveMetadataServiceTests.cs

```diff
@@ -29,17 +28,12 @@ namespace Microsoft.Health.Dicom.Tests.Integration.Features;
 
 public class RetrieveMetadataServiceTests : IClassFixture<DataStoreTestsFixture>
 {
-    private static readonly CancellationToken DefaultCancellationToken = new CancellationTokenSource().Token;
-
     private readonly RetrieveMetadataService _retrieveMetadataService;
     private readonly IInstanceStore _instanceStore;
     private readonly IMetadataStore _metadataStore;
     private readonly IETagGenerator _eTagGenerator;
     private readonly IDicomRequestContextAccessor _dicomRequestContextAccessor;
 
-    private readonly string _studyInstanceUid = TestUidGenerator.Generate();
```

Comment:
For these tests, the files are now based entirely on watermark and as such they cannot share the same value. Otherwise they will collide in the storage account.